infra/token-transfer-server/src/utils.js

```diff
@@ -0,0 +1,57 @@
+const Web3 = require('web3')
+
+const { Account } = require('./models')
+const { unlockDate } = require('./config')
+
+/**
+ * Allows use of async functions for an Express route.
+ */
+const asyncMiddleware = fn => (req, res, next) => {
+  Promise.resolve(fn(req, res, next)).catch(next)
+}
+
+const isEthereumAddress = value => {
+  if (!Web3.utils.isAddress(value)) {
+    throw new Error('Address is not a valid Ethereum address')
+  }
+  return true
+}
+
+const isExistingAddress = (value, { req }) => {
+  return Account.findOne({
+    where: {
+      userId: req.user.id,
+      address: value
+    }
+  }).then(account => {
+    if (account) {
+      throw new Error('Address already exists')
+    }
+  })
```

Comment:
super nit: return false rather than undefined
same comment for isExistingNickname function below.

Reply:
`true` I think you mean? express-validators only cares if it throws I think, `return false` would indicate it isn't valid.